project/plugins.sbt

```diff
@@ -2,20 +2,20 @@ logLevel := Level.Warn
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.6.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.13")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.25")
 
 // Code quality
 
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
-addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.2")
-addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.1")
-addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.3")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.7")
+addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.3")
```

Comment:
Probably not now, but getting off of scalariform, and on to scalafmt and scalafix should happen.

Reply:
Worth a ticket though.